source/features/latest-tag-button.tsx

```diff
@@ -13,6 +14,20 @@ import {groupButtons} from '../github-helpers/group-buttons';
 import getDefaultBranch from '../github-helpers/get-default-branch';
 import {buildRepoURL, getCurrentCommittish, getLatestVersionTag, getRepo} from '../github-helpers';
 
+export default async function addAfterBranchSelector(button: Element): Promise<void> {
```

Comment:
```suggestion
// eslist-disable......
export async function addAfterBranchSelector(button: Element): Promise<void> {
```
Nitpick
We dont do defaults from feature files